### PR TITLE
chore: add leading slash to internal hrefs in HTML tables

### LINKED ISSUE #27317: Broken link
### MDN URL

https://developer.mozilla.org/en-US/docs/Learn/HTML/Multimedia_and_embedding/Responsive_images

### What specific section or headline is this issue about?

Responsive images Page - Objective table at the top of the page

### What information was incorrect, unhelpful, or incomplete?

In the Objective section at the top of the page "`Learn how to use features like srcset and the <picture> element to implement responsive image solutions on websites.`" the link to srcset is broken 

Broken link pointing to https://developer.mozilla.org/en-US/docs/Learn/HTML/Multimedia_and_embedding/en-US/docs/Web/HTML/Element/img#srcset

### What did you expect to see?

Link to https://developer.mozilla.org/en-US/docs/Web/HTML/Element/img#srcset

### Do you have any supporting links, references, or citations?

_No response_

### Do you have anything more you want to share?

_No response_

### MDN metadata

<details>
<summary>Page report details</summary>

* Folder: `en-us/learn/html/multimedia_and_embedding/responsive_images`
* MDN URL: https://developer.mozilla.org/en-US/docs/Learn/HTML/Multimedia_and_embedding/Responsive_images
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/learn/html/multimedia_and_embedding/responsive_images/index.md
* Last commit: https://github.com/mdn/content/commit/a93bc54653c67cf504a08780b40dc15fdc904abb
* Document last modified: 2023-03-23T05:21:36.000Z

</details>

### PR DESCRIPTION
**Pull request by bsmth**
We have checks for markdown links that start with `/en` : https://github.com/mdn/content/blob/324bb0a68fb8c06c7d8cd2b5b9dee0f56ada9b23/.markdownlint-cli2.jsonc#L152-L156

This doesn't catch HTML tables. This PR does a search and replace for stray internal refs that start with `en-US` instead of `/en-US`.

### CODE REVIEW

**Comment by Josh-Cena** [comment]
I suggest changing the config to:

```json
 "search": "(?<=[(\"])en-US/docs",
 "replace": "/en-US/docs",
```

**Comment by bsmth** [comment]
> I suggest changing the config to:
> 
> ```json
>  "search": "(?<=[(\"])en-US/docs",
>  "replace": "/en-US/docs",
> ```

The CLI tool was erroring out with lookbehind assertion, maybe I'm missing something in how to write the rules in this config. Anyway I added an alternative in https://github.com/mdn/content/pull/27294/commits/4b3d3b91b1f02003f1000ed28e516a19561c61a9 that performs the replacement as expected.

File: .markdownlint-cli2.jsonc [hunk 1]
```diff
@@ -151,9 +151,9 @@
         {
           "name": "relative-link",
           "message": "Internal links should start with '/'",
-          "search": "(en-US/docs",
-          "replace": "(/en-US/docs",
-          "searchScope": "text"
+          "searchPattern": "/([\\(\"])en-US\\/docs/g",
+          "replace": "$1/en-US/docs",
+          "skipCode": true
```
Comment (bsmth) [inline .markdownlint-cli2.jsonc:156]: I think I may have overrun changes on main while resolving a merge conflict, I guess this should be `"searchScope": "text"`

Comment (Josh-Cena) [reply .markdownlint-cli2.jsonc:156]: I think so :)

Comment (bsmth) [reply .markdownlint-cli2.jsonc]: done in https://github.com/mdn/content/pull/27294/commits/59053126ab208c958c96caef1d017a2731787a50

**Review by caugner — APPROVED** [review]

**Comment by bsmth** [comment]
Thanks @caugner, merging 👍🏻

**Pull request merged**